source-control/pull-request-templates/README.md

```diff
@@ -0,0 +1,47 @@
+# Pull Request Templates
+
+To keep pull requests consistent and to help ensure quality across the project pull request templates are essential. Pull request templates can be used in both GitHub and Azure DevOps
+
+## GitHub
+
+In order to use a specific pull request template in GitHub you must include the specific template in your url. For example if you want to use `bug-template-md` you need to use query parameters when you open the PR, `https://github.com/[USERNAME OR ORG NAME]/[REPO NAME]/compare/master...[BRANCH NAME]?expand=1&template=bug-template.md`. If you want to have a default template so you do not have to always specificy the template in the URL make sure you have `PULL_REQUEST_TEMPLATE.md` in the root of your `.github` folder. Example below
+
+```plaintext
+- /.github
+    - /PULL_REQUEST_TEMPLATE
+        - bug-template.md
+        - feature-template.md
+    - PULL_REQUEST_TEMPLATE.md
+...
+```
+
+It is also worth noting that the rendering of markdown is different between Azure DevOps and GitHub so the links below are specific to the rendering within GitHub
+
```

Comment:
Maybe include a link to GitHub's PR template guidance at the end of this section.

https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

Reply:
good idea, added